src/index.js

```diff
@@ -102,59 +109,187 @@ const menu = async () => {
     }
   };
 
+  const battle = async (enemy, player) => {
+    console.log(`Your Agi: ${player.agi}\nBug Agi: ${enemy.agi}`);
+    if (player.agi >= enemy.agi) {
+      console.log("You attack first, you win!\n");
+    } else {
+      console.log("The bug attacks first, you lose!\n");
+    }
+  };
+
+  const quests = async (playerInfo) => {
+    clear();
+    let select = 1;
+    try {
+      while (select != 0) {
+        console.log("select a quest:\n");
+        let questList = await getApi("tasks");
```

Comment:
question: are the `tasks` or `bugs` changing at any point? I ask that because these api calls are inside a loop, and if the responses are the same every time, you could make the requests outside the loop